webpack/assets/javascripts/react_app/pages/common/PageLayout/components/ExportButton/ExportButton.test.js

```diff
@@ -0,0 +1,10 @@
+import { testComponentSnapshotsWithFixtures } from 'react-redux-test-utils';
+
+import ExportButton from './ExportButton';
+
+const fixtures = {
+  'render with minimal props': { url: 'url' },
```

Comment:
what about having another test with `text` and `title` props?

Reply:
Added, thanks!